src/main/java/racing/core/domain/RacingGame.java

```diff
@@ -0,0 +1,38 @@
+package racing.core.domain;
+
+import racing.core.dto.Trial;
+import racing.core.patterns.MoveStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+
+    public Cars cars;
+    public int numberOfTrials;
+
+    public RacingGame(String[] namesOfCars, int numberOfTrials) {
+        this.cars = makeUpEntry(namesOfCars);
+        this.numberOfTrials = numberOfTrials;
+    }
+
+    public List<Trial> run(MoveStrategy movement) {
+        List<Trial> trials = new ArrayList<>();
+        for (int i = 0; i < numberOfTrials; i++) {
+            trials.add(cars.nextTrial(movement));
+        }
+        return trials;
+    }
+
+    private Cars makeUpEntry(String[] namesOfCars) {
+        List<Car> carList = new ArrayList<>(namesOfCars.length);
+        for (String eachName : namesOfCars) {
+            carList.add(new Car(eachName));
+        }
```

Comment:
스트림을 사용해서 바꿔보았습니다:) 이렇게도 할 수 있다는 것이니, 참고만 해주세요!
```suggestion
        List<Car> carList = Arrays.stream(namesOfCars)
                .map(Car::new)
                .collect(Collectors.toList());
```